dmt/report/calculators.py

```diff
@@ -83,3 +85,51 @@ def calc(self):
             report = cursor.fetchall()
 
         return report
+
+
+def percentage_of(a, b):
+    return 100 * round(a / b, 2) if b else None
+
+
+class ProjectStatusCalculator(object):
+    def calc(self):
+        report = []
+
+        projects = Project.objects.filter(
+            ~Q(status='Defunct') & ~Q(status='Non-project')
```

Comment:
I have never seen `~` like this. What does it mean?

Reply:
This is one way to handle boolean sql statements in Django's ORM, like NOT, OR, AND, etc. https://docs.djangoproject.com/en/2.0/topics/db/queries/#complex-lookups-with-q-objects